pkg/bpf/map_linux_test.go

```diff
@@ -152,7 +152,7 @@ func TestPrivilegedOpenMap(t *testing.T) {
 
 	openedMap, err = OpenMap(MapPath(logger, "cilium_test"), &TestKey{}, &TestValue{})
 	require.NoError(t, err)
-	require.True(t, mapsEqual(openedMap, testMap))
+	mapsEqual(t, openedMap, testMap)
```

Comment:
```suggestion
	mapsEqual(t, testMap, openedMap)
```
To make sure the test output labels expected and actual values correctly in case something isn't equal.

Reply:
Done!